packages/captp/lib/captp.js

```diff
@@ -17,7 +17,7 @@ export { E };
 
 /**
  * @template T
- * @typedef {import('@agoric/eventual-send').ERef<T>} ERef
+ * @typedef {import('@agoric/promise-kit').ERef<T>} ERef
```

Comment:
Will this amount to replacing E() syntax with something else?

Reply:
No, that has not changed. The `eventual-send` on that line was simply stale. The `ERef` type is defined and exported by the `promise-kit` module. Has been for a long time.